docs/local-configuration.md

```diff
@@ -219,3 +219,21 @@ If your editor of choice happens to be Idea and you wnat to use its 'Run Dashboa
    ```
    discoveryLocations: http://eureka:password@discoveryInstance1:10022/eureka,http://eureka:password@discoveryInstance2:10033/eureka
    ```
+
+### Ports Used
```

Comment:
Reformat this`to markdown table ordered by port.

Reply:
Done